### ISSUE TITLE
Disable AirDrop broken

### ISSUE THREAD

**Issue by 0xmachos** [0xmachos/Lockdown#2]
Check command `defaults read com.apple.NetworkBrowser DisableAirDrop | grep 1` returns `The domain/default pair of (com.apple.NetworkBrowser, DisableAirDrop) does not exist`. 

`defaults read com.apple.NetworkBrowser` returns `Domain com.apple.NetworkBrowser does not exist`. 

`defaults read | grep "AirDrop"` returns

```
showAirDropImportAlert = 1;
            showAirDropImportUseFMFAppAlert = 0;
            showAirDropImportViewOniCloudAlert = 1;
            "com.apple.share.AirDrop.send",
        OneTimeAirDropReset = 1;
        "ShareRKServiceSystem_com_apple_share_AirDrop_send" =         {
```

None of those values look useful for disabling/ enabling AirDrop.

**Comment by 0xmachos** [comment]
[A partial solution](https://medium.com/@mariociabarra/wifried-ios-8-wifi-performance-issues-3029a164ce94):

`sudo ifconfig awdl0 down`

This brings down the Apple Wireless Direct Link (AWDL) interface which is also used for AirDrop. 

This doesn't persist over reboots and I'm unsure if the interface can be brought back up by the system.

**Comment by 0xmachos** [comment]
The key `DisableAirDrop ` in domain `com.apple.NetworkBrowser` does work to disable AirDrop. This only takes effect after `sharingd` has been restarted with `sudo killall -HUP sharingd`. 

This was addressed in c304a79ea39a60143b0158042a68664793f31f0c.
